androidapp/HopSquadApp/app/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
+  "project_info": {
+    "project_number": "205890785896",
+    "firebase_url": "https://hopsquadatx.firebaseio.com",
+    "project_id": "hopsquadatx",
```

Comment:
Is there a way to connect this project to the App Engine one? I was thinking about how to implement login there that's restricted to just one user. Should we keep app / web login all in one system or separate?

Reply:
Right now the app is connected to an app engine project that I created, so I think it shouldn't be that difficult to recreate the json file with the other app engine project in firebase.